internal/appsec/dyngo/instrumentation/grpcsec/tags.go

```diff
@@ -14,22 +14,20 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
 )
 
-// SetSecurityEventTags sets the AppSec-specific span tags when a security event
-// occurred into the service entry span.
-func SetSecurityEventTags(span ddtrace.Span, events []json.RawMessage, md map[string][]string) {
-	if err := setSecurityEventTags(span, events, md); err != nil {
-		log.Error("appsec: %v", err)
+// SetSecurityEventsTags sets the AppSec events span tags.
+func SetSecurityEventsTags(span ddtrace.Span, events []json.RawMessage) {
+	if err := setSecurityEventsTags(span, events); err != nil {
+		log.Error("appsec: unexpected error while creating the appsec events tags: %v", err)
 	}
 }
 
-func setSecurityEventTags(span ddtrace.Span, events []json.RawMessage, md map[string][]string) error {
-	if err := instrumentation.SetEventSpanTags(span, events); err != nil {
-		return err
-	}
+func setSecurityEventsTags(span ddtrace.Span, events []json.RawMessage) error {
+	return instrumentation.SetEventSpanTags(span, events)
```

Comment:
What's the point of wrapping this?

Reply:
- This one is used in the tests and returns an error for the assertions.
- And `SetSecurityEventsTags` is a helper that immediately logs the error for the two calls sites in our grpc instrum.